### PR TITLE
merge new template

### PR DESCRIPTION
**Pull request by ypriverol**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/quantms/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/quantms _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker` --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: assets/schema_input.json [hunk 1]
```diff
@@ -31,9 +31,6 @@
                 ]
             }
         },
-        "required": [
-            "sample",
-            "fastq_1"
-        ]
+        "required": ["sample", "fastq_1"]
```
Comment (jpfeuffer) [inline assets/schema_input.json:34]: Also looks like we don't need that file. Seems genomics specific.

File: docs/usage.md [hunk 2]
```diff
@@ -27,9 +27,9 @@ This will launch the pipeline with the `docker` configuration profile. See below
 Note that the pipeline will create the following files in your working directory:
 
 ```console
-work            # Directory containing the nextflow working files
-results         # Finished results (configurable, see below)
-.nextflow_log   # Log file from Nextflow
+work                # Directory containing the nextflow working files
+<OUTIDR>            # Finished results in specified location (defined with --outdir)
```
Comment (jpfeuffer) [inline docs/usage.md:31]:
```suggestion
<OUTDIR>            # Finished results in specified location (defined with --outdir)
```

File: docs/usage.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ This version number will be logged in reports when you run the pipeline, so that
...
+Several generic profiles are bundled with the pipeline which instruct the pipeline to use software packaged using different methods (Docker, Singularity, Podman, Shifter, Charliecloud, Conda) - see below. When using Biocontainers, most of these software packaging methods pull Docker containers from quay.io e.g [FastQC](https://quay.io/repository/biocontainers/fastqc) except for Singularity which directly downloads Singularity images via https hosted by the [Galaxy project](https://depot.galaxyproject.org/singularity/) and Conda which downloads and installs software locally from [Bioconda](https://bioconda.github.io/).
```
Comment (jpfeuffer) [inline docs/usage.md:60]: You could have left it. But I don't care much.

**Review by jpfeuffer — APPROVED** [review]

File: assets/schema_input.json (hunk 1 above)
Comment (ypriverol) [reply assets/schema_input.json]: Done. Here make more sense to do the review ;) Accepting suggestions is easier.

File: docs/usage.md (hunk 3 above)
Comment (ypriverol) [reply docs/usage.md:60]: Actually good point.

**Pull request closed without merging**